oak_runtime/src/introspection_browser_client/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Is the usage of this script documented anywhere? Also, in spite of reading the discussion on #1382 I am not clear on how this script will interoperate with the build system or whether this should be here or in the scripts directory.

Reply:
This script is used by NPM for the `npm run build` command, used to build the client. Commonly these are inlined in `package.json` (see the `dev` script), but since this one is a bit more complex it felt sensible to move it to its own file. 

Personally I'd argue against moving it into the scripts directory, since it isn't meant to be called directly by developers. 

I'd argue the integration of it into the build system is part of #1382, where it is called in the runner.

Reply:
I think `oak_runtime/src` is where we have all the rust code. I'd suggest `introspection_browser_client` to be moved to its own top-level folder. Is that possible?

Reply:
I agree that it feels odd to have non-rust code inside the `src` directory of a Rust crate. If possible, another directory at the same level as `src` seems more appropriate?

Reply:
Makes sense. :) I moved the folder up to live at the same level as `src`. I do think it's sensible to keep these files inside of the crate, since they are bundled by it.

https://github.com/project-oak/oak/blob/410c648a17de46838ff3e8a990f7cc0262677b74/oak_runtime/src/introspect.rs#L115